packages/slate-cli/store-configuration.md

```diff
@@ -0,0 +1,41 @@
+# Store / Environment Configuration Tips
+
+Knowing what to put in your `config.yml` isn't always straightforward.  This guide aims to clarify what data is needed, and where to get it.
+
+_Note: [ThemeKit](http://shopify.github.io/themekit/) is the tool that powers Slate deploys.  See it's
+[configuration variables documentation](http://shopify.github.io/themekit/configuration/) for more details._
+
+#### `config.yml` environment properties
+
+- **store:** the shopify-specifc URL for this store/ environment (ie. [my-store.myshopify.com](https://my-store.myshopify.com)).
+- **theme_id:** the unique id for the theme you want to write to when deploying to this store / environment
+- **password:** the password generated via a private app on this store / environment (necessary for API access)
+
+### Finding your theme_id
+
+_**warning:** some commands ([`start`](https://github.com/Shopify/slate-cli#start), [`deploy`](https://github.com/Shopify/slate-cli#deploy)) will overwrite the existing code on this `theme_id` with empty boilerplate.
```

Comment:
`([`start`](https://github.com/Shopify/slate/packages/slate-cli#start)` and ``([`deploy`](https://github.com/Shopify/slate/packages/slate-cli#deploy)`